### PR TITLE
docs: update lstm return type

### LINKED ISSUE #355: Fix LSTM Layer return type in docs
Currently the Axon v0.2.0 docs say the Axon.lstm/4 function returns
```
{{new_cell, new_hidden}, output_sequence}
```
but it actually returns this
```
{output_sequence, {new_cell, new_hidden}}
```

### PR DESCRIPTION
**Pull request by notjustinshaw**
fixes #355

### CODE REVIEW

**Review by notjustinshaw — COMMENTED** [review]
updated docs to match return type

**Comment by notjustinshaw** [comment]
fixes #355

File: lib/axon.ex [hunk 1]
```diff
@@ -2160,7 +2160,7 @@ defmodule Axon do
   LSTMs apply `Axon.Layers.lstm_cell/7` over an entire input
   sequence and return:
 
-      {{new_cell, new_hidden}, output_sequence}
```
Comment (notjustinshaw) [inline lib/axon.ex:2163]: here's the diff

**Review by polvalente — APPROVED** [review]

**Comment by polvalente** [comment]
Thanks!

**Pull request merged**